addons/knowledge/data/knowledge_data.xml

```diff
@@ -162,6 +162,15 @@
             <field name="name">Sales Playbook</field>
             <field name="icon">📘</field>
             <field name="body" type="html">
+                <div class="o_knowledge_toc o_knowledge_behavior_anchor o_knowledge_behavior_type_toc" contenteditable="false">
+                    <div class="o_knowledge_toc_content">
+                        <a data-oe-nodeid="0" contenteditable="false" href="#" class="oe_unremovable o_no_link_popover d-block o_knowledge_toc_link o_knowledge_toc_link_depth_0">Sales Playbook</a>
```

Comment:
I think it may be planned to later let end-users modify this block, which means they could add links that are not "ToC links". That class makes it easy to identify our custom links for the associated handler.

You might say it's a bit of "early optimization", but it doesn't hurt too much so I think I will keep it.